### PR TITLE
REPO-5215 Deprecated metadata extractors in 6.2.x

### PR DESCRIPTION
**Pull request by alandavis**

### CODE REVIEW

File: src/main/java/org/alfresco/repo/content/metadata/OfficeMetadataExtracter.java [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+/*
+ * Copyright (C) 2005 - 2020 Jesper Steen Møller
  *
  * This file is part of Alfresco
  *
```
Comment (SaraAspery) [inline src/main/java/org/alfresco/repo/content/metadata/OfficeMetadataExtracter.java:30]: Is this supposed to have 2 licenses?

File: src/test/java/org/alfresco/repo/action/executer/ContentMetadataExtracterTagMappingTest.java [hunk 2]
```diff
@@ -43,6 +43,7 @@
 import org.alfresco.repo.action.ActionModel;
 import org.alfresco.repo.action.AsynchronousActionExecutionQueuePolicies;
 import org.alfresco.repo.content.MimetypeMap;
+import org.alfresco.repo.content.metadata.AbstractMappingMetadataExtracter;
```
Comment (SaraAspery) [inline src/test/java/org/alfresco/repo/action/executer/ContentMetadataExtracterTagMappingTest.java:46]: unused import?

File: src/test/java/org/alfresco/repo/content/metadata/MP3MetadataExtracterTest.java [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+/*
+ * Copyright (C) 2005 - 2020 Jesper Steen Møller
  *
  * This file is part of Alfresco
  *
```
Comment (SaraAspery) [inline src/test/java/org/alfresco/repo/content/metadata/MP3MetadataExtracterTest.java:30]: 2 licenses

File: src/test/java/org/alfresco/repo/content/metadata/TikaAudioMetadataExtracterTest.java [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+/*
+ * Copyright (C) 2005 - 2020 Jesper Steen Møller
  *
  * This file is part of Alfresco
  *
```
Comment (SaraAspery) [inline src/test/java/org/alfresco/repo/content/metadata/TikaAudioMetadataExtracterTest.java:30]: 2 licenses

File: src/main/java/org/alfresco/repo/content/metadata/OfficeMetadataExtracter.java (hunk 1 above)
Comment (alandavis) [reply src/main/java/org/alfresco/repo/content/metadata/OfficeMetadataExtracter.java:30]: Yes two headers!

File: src/test/java/org/alfresco/repo/action/executer/ContentMetadataExtracterTagMappingTest.java (hunk 2 above)
Comment (alandavis) [reply src/test/java/org/alfresco/repo/action/executer/ContentMetadataExtracterTagMappingTest.java:46]: Removed. Thanks

File: src/main/java/org/alfresco/repo/content/metadata/OfficeMetadataExtracter.java (hunk 1 above)
Comment (alandavis) [reply src/main/java/org/alfresco/repo/content/metadata/OfficeMetadataExtracter.java:30]: Yes, I think there was a special agreement with this person. Hence an alfresco and personal copyright.

File: src/test/java/org/alfresco/repo/content/metadata/MP3MetadataExtracterTest.java (hunk 3 above)
Comment (alandavis) [reply src/test/java/org/alfresco/repo/content/metadata/MP3MetadataExtracterTest.java:30]: See above.

File: src/test/java/org/alfresco/repo/content/metadata/TikaAudioMetadataExtracterTest.java (hunk 4 above)
Comment (alandavis) [reply src/test/java/org/alfresco/repo/content/metadata/TikaAudioMetadataExtracterTest.java:30]: See above.

**Pull request merged**